tools/bumper/component_commands.go

```diff
@@ -67,10 +67,28 @@ type gitRepo struct {
 }
 
 const (
-	cnaoOwner = "kubevirt"
-	cnaoRepo  = "cluster-network-addons-operator"
+	bumpingRepoUrl = "https://github.com/kubevirt/cluster-network-addons-operator"
```

Comment:
we are not going to create new clone we don't need this

Reply:
now that cnao is kept in component struct, we can use `getComponentNameFromUrl`, `getComponentOwnerFromUrl` to get `cnaoOwner`, `cnaoRepo` from there. keeping this in url makes it more robust.